src/OrchardCore.Modules/OrchardCore.Roles/Views/Items/RemoveUserRoleTask.Fields.Edit.cshtml

```diff
@@ -0,0 +1,16 @@
+@using OrchardCore.Roles.Workflows.ViewModels
+@model RemoveUserRoleTaskViewModel
+
+<div class="mb-3" asp-validation-class-for="UserName">
+    <label asp-for="UserName">@T["UserName"]</label>
+    <input type="text" asp-for="UserName" class="form-control code" />
+    <span asp-validation-for="UserName"></span>
+    <span class="hint">@T["The User to update. With Liquid support."]</span>
+</div>
+
+<div class="mb-3" asp-validation-class-for="RoleName">
+    <label asp-for="RoleName">@T["RoleName"]</label>
```

Comment:
Why are you not using RoleEntry?

Reply:
Yes and I suggested that you don't need Liquid text in this case. 

Can you think of an example where you would need to use Liquid to identify a list of role names?